source/data/FluxStore.js

```diff
@@ -0,0 +1,190 @@
+(function (enyo, scope) {
+
+	var kind = enyo.kind,
+		EventEmitter = enyo.EventEmitter,
+		StateSupport = enyo.StateSupport;
+
+	function update(node, src) {
+		var key, val;
+
+		if (!(node && src)) return;
+
+		// avoid iterating over null/undefined
+		for (key in src) {
+			// avoid looking this value up over and over
+			val = src[key];
+			// don't clear current values if one wasn't returned
+			if (val != null) {
+				// if there isn't a value already we short-circuit the object test for efficiency
+				if (!node[key] || !enyo.isObject(val)) node[key] = val;
+				else update(node[key], val);
+			}
+		}
+
+		return node;
+	}
+
+	/**
+	*
+    *
+	* @class enyo.FluxStore
+	* @mixes enyo.EventEmitter
+	* @extends enyo.Object
+	* @public
+	*/
+	kind(
+		/** @lends enyo.FluxStore.prototype */ {
+		name: 'enyo.FluxStore',
+
+		/**
+		* @private
+		*/
+		kind: enyo.Object,
+
+
+		/**
+		* @private
+		*/
+		data: {},
```

Comment:
Should this be initialized in `create` instead, to prevent object reference sharing?